website/src/client/components/App.tsx

```diff
@@ -342,7 +344,7 @@ class Main extends React.Component<Props, State> {
     }
 
     Analytics.getInstance().setCommonData({
-      snackId: this.state.session.id,
+      snackId: this.state.session.uuid,
```

Comment:
I think i prefer to stick with `snackId` over anything else because it most accurately describes what this field is. i can run a query similar to `where snackId != {uuidRegex}` and clear out the ones that return false.

Reply:
oh my bad, that was an old comment, skip that pls 😄